styles/Global.css

```diff
@@ -47,11 +49,61 @@ a:hover {
 	background-color: transparent;
 	border: none;
 }
+.index--ft__wrapper {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	align-items: center;
+	text-indent: 8px;
+	color: #ff455b;
+	font-family: 'Mulish';
+	text-transform: uppercase;
+	font-size: 12px;
+	margin-bottom: -45px;
+}
+.index--ul {
+	margin-top: 39vh;
+	display: flex;
+	justify-content: space-around;
+	flex-direction: row;
+	list-style: none;
+	color: #989cac;
+}
+.p--resume,
+.p--blog,
+.p--about {
+	font-family: mulish;
+	color: #989cac;
+	font-size: 20px;
+	margin: 0;
+}
+.index--first {
```

Comment:
You're still in the index section the `p` blocks should not be in the middle of the index block rules.